manuscript/Cycle-Notation.md

```diff
@@ -29,7 +29,7 @@ leave out numbers that are not permuted. In our example we end up with the
 following cycle notation.
 
 ```plain
-(0 1 2)(3 4)
+(0 2 1)(3 4)
 ```
```

Comment:
Ah ok, now I get it. 
I think I was put off by the phase 

> We list 0 and see **where it is** permuted to.

So, I was thinking positional instead of transitional. If it had said
> We list 0 and see **what it has** permuted to.

I _might_ have gotten it earlier :)

I'll amend the pull request!